config.py

```
@@ -58,17 +58,183 @@ class DeviceMeta(TypedDict):
             "IN": "CPH2573", 
             "CN": "PJD110"
         }
+    },
+    "12R": {
+        "name": "OnePlus 12R",
+        "models": {
+            "GLO": "CPH2585",
+            "EU": "CPH2609",
+            "IN": "CPH2585",
+            "IN": "CPH2585"
```

Ruff (0.14.14)

[error] Dictionary key literal `"IN"` repeated

Remove repeated key literal `"IN"`

(F601)